building/build-debs/homeworld-admin-tools/src/infra.py

```diff
@@ -5,6 +5,9 @@
 
 
 def infra_admit(server_principal: str) -> None:
+    config = configuration.get_config()
+    if not config.has_node(server_principal):
+        command.fail("Server %s not found in project configuration" % server_principal)
```

Comment:
Have you tested this? I can't remember exactly, but I seem to remember that `has_node` uses unqualified hostnames, whereas this method needs fully qualified hostnames.

Reply:
I think I need the real setup.yml to test it with real nodes, otherwise can I still only try `admit`

Reply:
You should be able to access the real setup.yaml under https://github.mit.edu/sipb/hyades-cluster

Reply:
Also, you'll need access to the cluster. Do you have a /root instance?